### ISSUE TITLE
Error with HA 2021.5.0

### ISSUE THREAD

**Issue by PierreScerri** [pnbruckner/ha-sun2#38]
After upgrading to HA 2021.5.0 I get this error:

```
This error originated from a custom integration.

Logger: homeassistant.config
Source: custom_components/sun2/sensor.py:5
Integration: sun2 (documentation, issues)
First occurred: 9:41:17 AM (2 occurrences)
Last logged: 9:45:38 AM

Platform error: sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config.py", line 853, in async_process_component_config
    platform = p_integration.get_platform(domain)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 493, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 498, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/config/custom_components/sun2/sensor.py", line 5, in <module>
    from astral import AstralError
ImportError: cannot import name 'AstralError' from 'astral' (/usr/local/lib/python3.8/site-packages/astral/__init__.py)

```

**Comment by MizterB** [comment]
This is due to an astral upgrade in core HA:
https://developers.home-assistant.io/blog/2021/04/09/astral-2.2/

**Comment by MizterB** [comment]
AstralError was removed in this commit
https://github.com/sffjunkie/astral/commit/e036fdc291ba46a79d7cc7cae5b82574d874d4fc

In `sensor.py`, I commented out the AstralError import and and replaces its use in` _get_astral_event` with ValueError.  Seems to be working now.

**Comment by PierreScerri** [comment]
Updated to 2.0.0
No Errors so far.

**Comment by pnbruckner** [comment]
@MizterB @PierreScerri 

This was taken care of with release 1.8.3, which was done almost a month before HA released 2021.5.x (where the astral package was updated.)
